### DISCUSSION TITLE
Confirmation to Add Fourth Icon to Header

### DISCUSSION THREAD

**Discussion by Sa7aR-Kassem** [TDRA-ae/aegov-dls#80] [General]
**Description:**
We plan to update the header by adding a fourth icon alongside the current icons. This change is intended to improve user experience and make the header more functional.

**Proposed Changes:**
Add a new icon in the header next to the existing ones.

**Request for Confirmation:**
Please confirm if we can proceed with adding the fourth icon to the header.

**Comment by moiz-conceptlz** [comment]
@Sa7aR-Kassem The same on the right on the navigation is a holder for icons. You may add or remove icons - however it must be relevant and not disrupt the layout on all screen sizes. As long as adding an additional icon does not cause any break on a layout, is handled on mobile and is relevant to be placed on a global level - I dont see any harm in doing so.
